.travis.yml

```diff
@@ -40,3 +40,8 @@ notifications:
 branches:
   except:
     - /^*-v[0-9]/
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then export REPO=mayadataio/dmaas-operator-amd64;
```

Comment:
intentation also need to be corrected

Reply:
instead of looking for TRAVIS_PULL_REQUEST flag, can we just check that credentials (DOCKER_USER and DOCKER_PASS) are provided or not?